README.md

```diff
@@ -1,26 +1,45 @@
 ## Description
-These rules refer to 
+These rules refer to http://developer.android.com/guide/developing/device.html and include many suggestions from the Archlinux and Github Community.
 
-http://developer.android.com/guide/developing/device.html
+If you're developing on Ubuntu Linux, you need to add a udev rules file that contains a USB configuration for each type of device you want to use for development. In the rules file, each device manufacturer is identified by a unique vendor ID, as specified by the ATTR{idVendor} property. For a list of vendor IDs, see USB Vendor IDs, below.
 
-and include many suggestions from the Archlinux and Github Community
+Use this format to add each vendor to the file(example vendor ID is for HTC):  
+`SUBSYSTEM=="usb", ATTR{idVendor}=="0bb4", MODE="0666", GROUP="plugdev"`
+
+The MODE assignment specifies read/write permissions, and GROUP defines which Unix group owns the device node.
+
```

Comment:
Also in multi user instances OR under-privileged systems (requiring sudo) group membership to plugdev is  needed in some distros.   The edits here seem to be  aimed at a more  novice 010 user / dev  and  some more advanced conventions seem randomly  disregarded.  Let's line item comment them as they appear or  at the top of this file or  somewhere in the repo have a 'Used / Assumed Conventions.md'   file to alleviate possible new dev /user confusions.